compiler/codegen/symbol.cpp

```diff
@@ -2282,27 +2272,6 @@ GenRet FnSymbol::codegenPYXType() {
       if (origt == dtString) {
         returnStmt += "\tret = ret.decode(\'utf-8\')\n";  
       }
-
-    } else if (retType == dtStringC) {
-
-      //
-      // NOTE: The result of the routine call is a `c_string`, AKA const
-      // char*. We know that the only thing we could be returning here
-      // is a `c_string` (since Chapel strings are returned as
-      // "chpl_byte_buffer"), which means that we can't call `chpl_free` on
-      // it because we know nothing about its lifetime.
-      //
-      funcCall += "cdef const char* rdata = <char*> ";
-
-      // Get the length of the "char*" using strlen.
-      returnStmt += "\tcdef Py_ssize_t rsize = strlen(rdata)\n";
-
-      // Create a new Python bytes that is a copy of the Chapel string.
-      returnStmt += "\tret = PyBytes_FromStringAndSize(rdata, rsize)\n";
```

Comment:
Does this mean we no longer need to include it in the generated cython file?

Reply:
No, not at this point, but it does seem like something we're going to have to include eventually, no? (I'm not losing sleep over it.)

Reply:
Okay.  Not the end of the world to include it, then!